### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by andyexeter**
"Secret" should be plural "Secrets" in this context.

### CODE REVIEW

File: best_practices.rst [hunk 1]
```diff
@@ -88,7 +88,7 @@ application behavior.
 :ref:`Use env vars in your project <config-env-vars>` to define these options
 and create multiple ``.env`` files to :ref:`configure env vars per environment <config-dot-env>`.
 
-Use Secret for Sensitive Information
+Use Secrets for Sensitive Information
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (OskarStark) [inline best_practices.rst]: Please also increase the length of the underline

Comment (OskarStark) [reply best_practices.rst]: @javiereguiluz we also need to keep the old link/deeplink working, can you please share an example, I am currently on a phone

**Comment by alexislefebvre** [comment]
This title exists in Symfony 4.4 https://symfony.com/doc/4.4/best_practices.html#use-secret-for-sensitive-information I think the PR should target 4.4.

File: best_practices.rst (hunk 1 above)
Comment (xabbuh) [inline best_practices.rst]:
```php
.. _use-secret-for-sensitive-information:

Use Secrets for Sensitive Information
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I think this should work.

Comment (OskarStark) [reply best_practices.rst]: Thanks, that's it

**Comment by andyexeter** [comment]
What's the best way of updating the PR to target 4.4? I tried to change the base via github but it brought across hundreds of other commits.

**Comment by alexislefebvre** [comment]
@andyexeter 

- easiest method: close this PR, edit the [file from the 4.4 branch](https://github.com/symfony/symfony-docs/blob/4.4/best_practices.rst) and open a PR targeting 4.4
- more complicated method: rebasing may be too much work, but you can checkout the 4.4 branch locally, cherry-pick your commit, then push force on the branch of this PR, and finally change the targeted branch in this PR

**Comment by OskarStark** [comment]
We can merge it against 4.4 no problem 😃

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @andyexeter.

**Pull request merged**